administrator/components/com_media/layouts/toolbar/create-folder.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_media
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+defined('_JEXEC') or die;
+$title = JText::_('COM_MEDIA_CREATE_NEW_FOLDER');
+?>
+<button class="btn btn-sm btn-outline-info" onclick="MediaManager.Event.$emit('onClickCreateFolder');">
+    <span class="icon- fa fa-folder" title="<?php echo $title; ?>"></span> <?php echo $title; ?>
```

Comment:
<i> isn’t the icon tag, it’s the italic tag, and it doesn’t make much difference which you pick. 
Dont know who the **** came up with the idea to use <i>s, but it is semantically not correct.

<span> is more semantically correct, because it has no semantics.

And regarding icon- class: it is currently needed by the admin theme to render buttons in that style.